ci-operator/config/tnozicka/openshift-acme/master.json

```diff
@@ -0,0 +1,60 @@
+{
+  "base_images": {
+    "os": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "centos",
+      "tag": "7"
+    }
+  },
+  "test_base_image": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "release",
+    "tag": "golang-1.10"
+  },
+
+  "canonical_go_repository": "github.com/tnozicka/openshift-acme",
+
+  "binary_build_commands": "make build GOFLAGS='-v'",
+  "test_binary_build_commands": "make build GOFLAGS='-v -race'",
+
+  "images": [
+    {
+      "from": "os",
+      "to": "openshift-acme",
+      "dockerfile_path": "images/Dockerfile",
+      "inputs": {
+        "bin": {
+          "as": ["bin"]
+        }
+      }
+    }
+  ],
+
+  "tests": [
+    {
+      "as": "verify",
+      "from": "bin",
+      "commands": "mkdir /tmp/artifacts && make verify -k |& tee /tmp/artifacts/verify.log",
+      "artifact_dir": "/tmp/artifacts"
```

Comment:
I'd have hoped this actually creates that directory, but it doesn't.

Reply:
Yeah we need to fix this internally